inc/spbc-settings.php

```diff
@@ -6428,3 +6441,66 @@ function spbc_settings_draw_save_button_bottom_block()
         __('Back to navigation', 'security-malware-firewall')
     );
 }
+
+/**
+ * Get HTML content for change role template
+ * @return void *echo
+ */
+function spbc_change_role_template()
```

Comment:
No Unit tests for this logic at all.

Reply:
Thanks, fixed https://github.com/CleanTalk/security-malware-firewall/pull/513/commits/d1a0e00780dc0fe7e3da80a396fb41ba66d97c08